openrpc.json

```diff
@@ -511,6 +511,36 @@
         }
       ]
     },
+    {
+      "name": "wallet_scanQRCode",
+      "tags": [
+        {
+          "$ref": "#/components/tags/MetaMask"
+        },
+        {
+          "$ref": "#/components/tags/Mobile"
+        }
+      ],
+      "summary": "Requests that the user scan a QR code.",
+      "description": "Requests that the user scan a QR code using their device camera. Introduced by [EIP-945](https://github.com/ethereum/EIPs/issues/945).",
+      "params": [
+        {
+          "name": "regex",
```

Comment:
Is it a set of regex(S) or is it expecting a single regex?

ie, does the method call look like:

```
method: 'wallet_scanQRCode',
params: [ [ 'regexGoesHere', 'anotherRegexGoesHere'  ]  ]
```

or 

```
method: 'wallet_scanQRCode',
params: [ 'anotherRegexGoesHere'  ]
```


or 

```
method: 'wallet_scanQRCode',
params: [ [ 'anotherRegexGoesHere' ]  ]
```

Reply:
I believe #2 according to the manual reference: https://docs.metamask.io/wallet/reference/rpc-api/#wallet_scanqrcode